apps/script/HoC2023ScriptFiles/HoC2023AiHelperFunctions.py

```diff
@@ -14,15 +14,15 @@
 module_path = ".."
 sys.path.append(os.path.abspath(module_path))
 
-# KNOWN ISSUE: Their version of utils was a custom library file unique to their bedrock tutorial located here: https://github.com/aws-samples/amazon-bedrock-workshop/blob/109ed616fd14c9eb26eda9bef96eb78c490d5ef6/utils/bedrock.py
-# While it WAS working before, it doesn't work now and the code contains within the utils folder will need to be migrated out to get this working locally
-from utils import bedrock, print_ww
+# aws_utils contains a custom library file unique to their bedrock tutorial located here: https://github.com/aws-samples/amazon-bedrock-workshop/blob/109ed616fd14c9eb26eda9bef96eb78c490d5ef6/utils/bedrock.py
+# See readme.md for more detailed access information and further amazon documentation links
+from aws_utils import bedrock
 
 # ---- ⚠️ Un-comment and edit the below lines as needed for your AWS setup ⚠️ ----
 
-# os.environ["AWS_DEFAULT_REGION"] = "<REGION_NAME>"  # E.g. "us-east-1"
-# os.environ["AWS_PROFILE"] = "<YOUR_PROFILE>"
-# os.environ["BEDROCK_ASSUME_ROLE"] = "<YOUR_ROLE_ARN>"  # E.g. "arn:aws:..."
+# os.environ["AWS_DEFAULT_REGION"] = "us-east-1"  # E.g. "us-east-1"
+# os.environ["AWS_PROFILE"] = "codeorg-dev"
+# os.environ["BEDROCK_ASSUME_ROLE"] = "arn:aws:iam::165336972514:user/Tyrone"  # E.g. "arn:aws:..."
```

Comment:
should these be updated to the actual values? Or should they still be the placeholder defaults?

Reply:
I think both default_region and assume_role are super user-specific and so would need to be updated to whoever was trying to run the script on their personal device. These placeholder values are definitely just there to illustrate the format of what they would need.